qutebrowser/config/config.py

```diff
@@ -523,6 +525,13 @@ def set_command(self, win_id: {'special': 'win_id'},
                 val = self.get(sectname, optname[:-1], transformed=False)
                 message.info(win_id, "{} {} = {}".format(
                     sectname, optname[:-1], val), immediately=True)
+            elif optname.endswith('!'):
+                val = self.get(sectname, optname[:-1])
+                layer = 'temp' if temp else 'conf'
+                if type(val) == type(True):
```

Comment:
This should be written as `isinstance(val, bool)` instead. See http://docs.quantifiedcode.com/python-code-patterns/readability/do_not_compare_types_use_isinstance.html for example.